ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/RD-RT-ADMIN-SUBFIELD-L3LEAF6.yml

```diff
@@ -1,10 +1,8 @@
 ---
-
 overlay_rd_type:
-# Expected result: rd 65535:*
-# Testing inline jinja with integer values
-  admin_subfield: "{{ non_existing_variable | default(65525) }}"
-  admin_subfield_offset: "{{ non_existing_variable | default(10) }}"
+  # Expected result: rd 65535:*
+  admin_subfield: 65525
+  admin_subfield_offset: 10
```

Comment:
we may not need this host at all since this now test regular integers which I believe are tested elsewhere.

Reply:
keeping it to have combination of integer + offset